tests/route/test_route_flow_counter.py

```diff
@@ -214,4 +214,4 @@ def _get_nexthop(self, duthost, ipv6):
         else:
             cmd = 'show ip bgp summary'
         parse_result = duthost.show_and_parse(cmd)
-        return parse_result[0]['neighbhor']
```

Comment:
The logic need to be compatible between 'Neighbhor' and 'neighbor'.
Since there's a long time before the new image got built.
If we test the old image with the new changed test code, it will fail and affect the passing rating/